Facebook.Android/source/Xamarin.Facebook.sln

```diff
@@ -1,95 +1,95 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 11.00

```

Comment:
Moved a copy with just the source into the source folder so we can just build and pack that instead of worrying about samples.